public/scripts/main.js

```diff
@@ -0,0 +1 @@
+$(edit-task).on('click', () => {})
```

Comment:
This has no params and no return. Did you mean to commit this? Is it just adding an onClick Listener?